tests/test_bug_features.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import json
+import os
+
+from bugbug.bug_features import component
+from bugbug.bug_features import has_crash_signature
+from bugbug.bug_features import has_github_url
+from bugbug.bug_features import has_regression_range
+from bugbug.bug_features import has_str
+from bugbug.bug_features import has_url
+from bugbug.bug_features import has_w3c_url
+from bugbug.bug_features import is_coverity_issue
+from bugbug.bug_features import keywords
+from bugbug.bug_features import landings
+from bugbug.bug_features import patches
+from bugbug.bug_features import product
+from bugbug.bug_features import severity
+from bugbug.bug_features import title
+from bugbug.bug_features import whiteboard
+
+
+def read(path, function):
```

Comment:
`function` is actually a class, so I'd call it `cls` or `feature_extraction_cls`.

Reply:
And `check` should be `feature_extractor`.